include/rocksdb/c.h

```diff
@@ -737,6 +737,8 @@ rocksdb_block_based_options_set_pin_top_level_index_and_filter(
     rocksdb_block_based_table_options_t*, unsigned char);
 extern ROCKSDB_LIBRARY_API void rocksdb_options_set_block_based_table_factory(
     rocksdb_options_t* opt, rocksdb_block_based_table_options_t* table_options);
+extern ROCKSDB_LIBRARY_API void rocksdb_options_set_atomic_flush(
```

Comment:
Thanks for contributing @mappum !

The patch looks good to me, just one comment: could you move this declaration to right below `rocksdb_options_set_ratelimiter` so the order is the same in the header and the `.cc` file?

Reply:
:heavy_check_mark: Updated as requested.